File: src/main/java/com/huanletao/huanletao/service/api/CategoryService.java
package com.huanletao.huanletao.service.api;

import com.huanletao.huanletao.entity.WebCategory;

import java.util.List;

/**
 * Created with IntelliJ IDEA.
 *
 * @auther: huangjianfeng
 * @Date: 2019/12/28
 * @Time: 11:30
 * Description:
 */
public interface CategoryService {

    //查询所有数据、
    List<WebCategory> findAll();

    //保存数据
    void save(WebCategory webCategory);

    //删除分类
    void delete(int[] ids);
}

File: system/src/main/webapp/static/js/services/categoryService.js
//服务层
app.service('categoryService',function($http){

    //请求后台查询所有数据信息。
    this.findAll = function () {
        return $http.get("../cate/findAll.do")
    }

    //将数据保存到数据库
    this.save = function (entity) {
        return $http.post("../cate/save.do",entity);
    }

    //删除指定的分类信息。
    this.deleteCate = function (ids) {
        return $http.get("../cate/delete.do?ids="+ids);
    }



});
